lnwallet/rpcwallet/rpcwallet.go

```diff
@@ -585,6 +586,79 @@ func (r *RPCKeyRing) remoteSign(tx *wire.MsgTx, signDesc *input.SignDescriptor,
 	in := &packet.Inputs[signDesc.InputIndex]
 	txIn := tx.TxIn[signDesc.InputIndex]
 
+	// Things are a bit tricky with the sign descriptor. There basically are
```

Comment:
Great contextual comment here: 👍

I wish protobuf still had that "default" option which would let us better handle (legacy) edge cases like this where we need to distinguish an actual zero value vs the field not being set at all...